cmake/config/template-arguments.in

```diff
@@ -32,29 +38,46 @@ MPI_SCALARS     := { int;
                      unsigned long long int;
                      float;
                      double;
-                     long double }
+                     long double; 
+                     std::complex<double>;
```

Comment:
These belong in some expansion list other than `DIFFERENTIABLE_REAL_SCALARS`.

BTW. Thanks for all of the work that you're doing on this!

Reply:
I don't think I understand whether you want me to change anything here?

Reply:
I don't know how old the branch that you're working off of is, but I believe that @luca-heltai and I made [some alterations here a while ago](https://github.com/dealii/dealii/commit/f8b5289cf867e9493dfb04ee156d4d30577bd6b0) so I think that these changes are redundant or conflicting. So I think that these changes should be reverted. Here's the current state of these lines on `master`: 
```
// differentiable scalar types
DIFFERENTIABLE_REAL_SCALARS := {
		    @DEAL_II_EXPAND_TRILINOS_SACADO_TYPES@;
		    } 

// all possible scalar types
ALL_SCALAR_TYPES := {
    double; float ; std::complex<double>; std::complex<float>;
    @DEAL_II_EXPAND_TRILINOS_SACADO_TYPES@;
}
```

Reply:
Ah, drats. Sorry, I missed some code folding in github. Please ignore!